src/batch-client/index.ts

```diff
@@ -783,6 +784,13 @@ export class ZimbraBatchClient {
 			singleRequest: true
 		}).then(Boolean);
 
+	public distributionListAction = (dlActions: DistributionListActionInput) =>
+		this.jsonRequest({
+			name: 'DistributionListAction',
+			body: dlActions,
+			namespace: Namespace.Account
+		}).then(Boolean);
```

Comment:
why do we need `.then(Boolean)` ?

Reply:
I am getting below response from 'DistributionListAction' api. Hence passing boolean 
<img width="759" alt="image" src="https://github.com/Zimbra/zm-api-js-client/assets/131649520/d4fc12e6-9998-406a-a4da-c48d5d2d2b75">

Reply:
so how does passing `Boolean` help in this case? If we don't care about the response then we should not attach the `.then()` handler.

Reply:
@HimanshuSojitraSynacor, If I removed the `.then(Boolen)` and set the return type of api to `String` like `distributionListAction(dlActions: DistributionListActionInput!): String` then I get below error.
<img width="693" alt="image" src="https://github.com/Zimbra/zm-api-js-client/assets/131649520/e65ee9da-2e1a-4171-b719-42a40c92b0fa">